src/sharepoint/account/ACCOUNT.html

```diff
@@ -41,105 +41,73 @@
+                    <oui-column data-title="tr('sharepoint_accounts_one_drive_space')">
+                        <span data-ng-if="!$row.activated">-</span>
+                        <div class="oui-progress m-0" style="min-width:140px"
+                             data-ng-class="{
+                                'oui-progress_success': $row.percentUse < 51,
+                                'oui-progress_warning': $row.percentUse > 50 && $row.percentUse < 81,
+                                'oui-progress_error': $row.percentUse > 80
+                            }"
+                             data-ng-if="$row.activated">
+                            <div class="oui-progress__bar oui-progress__bar_text-left" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="{{$row.percentUse}}"
+                                 data-ng-class="{
+                                    'oui-progress__bar_success': $row.percentUse < 51,
+                                    'oui-progress__bar_warning': $row.percentUse > 50 && $row.percentUse < 81,
+                                    'oui-progress__bar_error': $row.percentUse > 80
+                                 }"
+                                 data-ng-style="{'width': $row.percentUse + '%'}">
+                                <span class="oui-progress__label"
+                                      data-ng-bind="::$row.usedQuota.value + ' ' + tr('unit_size_' + $row.usedQuota.symbol) + ' / ' + $row.totalQuota.value + ' ' + tr('unit_size_' + $row.totalQuota.symbol)"></span>
+                            </div>
+                        </div>
+                    </oui-column>
+                    <oui-column data-title="tr('sharepoint_accounts_administrator')" data-property="accessRights">
+                        <span class="oui-icon oui-icon-success text-success" aria-label="{{i18n.common_yes}}"
+                              data-ng-if="$row.accessRights === 'administrator'" ></span>
+                        <span data-ng-if="$row.accessRights !== 'administrator'"> - </span>
+                    </oui-column>
+                    <oui-column data-title="tr('sharepoint_accounts_user')" data-property="accessRights">
+                        <span class="oui-icon oui-icon-success text-success" aria-label="{{i18n.common_yes}}"
+                              data-ng-if="$row.accessRights"></span>
+                        <span data-ng-if="!$row.accessRights"> - </span>
+                    </oui-column>
+                    <oui-column data-title="tr('sharepoint_accounts_license_office')" data-property="officeLicense">
+                        <span class="oui-icon oui-icon-success text-success" aria-label="{{i18n.common_yes}}"
+                              data-ng-if="$row.officeLicense"></span>
+                        <span data-ng-if="!$row.officeLicense"> - </span>
+                    </oui-column>
+                    <oui-action-menu  data-align="end" data-compact>
```

Comment:
Very minor, there are 2 spaces there. Doesn't this module have eslint installed ?